changelog.d/warnings/2929.enhance.rst.txt

```diff
@@ -0,0 +1,2 @@
+Added features to mod to be able to toggle warns being sent to both DM's and channel.
+Added a command to set the channel the warns get sent to. (defaults to the channel the warn was issued in). 
```

Comment:
This file should be called `2929.enhance.rst` not `2929.enhance.rst.txt`

Reply:
I saw you marked this as resolved, but from the looks of it you renamed wrong file ;)